apiv2/project/project_errors.go

```diff
@@ -217,6 +278,55 @@ func (e *ErrProjectUnknownResource) Error() string {
 	return ErrProjectUnknownResourceMsg
 }
 
+// retrieveMetadataValue returns the value of the metadata k that is contained in the project metadata m.
+// Returns an empty string plus an error when encountering a nil pointer, or if the requested key k is invalid.
+func retrieveMetadataValue(k MetadataKey, m *modelv2.ProjectMetadata) (string, error) {
+	var r string
+	var sPtr *string
+
+	switch k {
+	case ProjectMetadataKeyEnableContentTrust:
+		if m.EnableContentTrust == sPtr {
```

Comment:
Shouldn't `== nil` work just as well? What's `sPtr` for, precisely?

Reply:
Spot on - `sPtr` is obsolete and using `nil` for comparison is way more convenient. My bad, Thanks!